src/index.js

```diff
@@ -14,6 +15,12 @@ const mergePackageJson = require('merge-package.json');
 const gitDiffApply = require('git-diff-apply');
 const run = require('./run');
 const utils = require('./utils');
+const pkg = require('../package.json');
+
+updateNotifier({
```

Comment:
Can we move this to the bin/ember-cli-update.js file please? We probably shouldn't print this if used as a library vs an executable.

Reply:
That makes sense, will make the change.